slasher/db/proposer_slashings_test.go

```diff
@@ -133,3 +134,30 @@ func TestStore_UpdateProposerSlashingStatus(t *testing.T) {
 	}
 
 }
+
+func TestStore_SaveProposerSlashings(t *testing.T) {
+	app := cli.NewApp()
+	set := flag.NewFlagSet("test", 0)
+	ctx := cli.NewContext(app, set, nil)
+	db := SetupSlasherDB(t, ctx)
+	defer TeardownSlasherDB(t, db)
+	ps := []*ethpb.ProposerSlashing{
+		&ethpb.ProposerSlashing{ProposerIndex: 1},
+		&ethpb.ProposerSlashing{ProposerIndex: 2},
+		&ethpb.ProposerSlashing{ProposerIndex: 3},
+	}
+	err := db.SaveProposeerSlashings(Active, ps)
+	if err != nil {
+		t.Fatalf("save proposer slashings failed: %v", err)
```

Comment:
same here regarding capitalization

Reply:
thanks, fixed all capitalization in those files